Userland/Libraries/LibJS/Tests/test-common.js

```diff
@@ -353,7 +353,12 @@ class ExpectationError extends Error {
         toEval() {
             this.__expect(typeof this.target === "string");
             const success = canParseSource(this.target);
-            this.__expect(this.inverted ? !success : success);
+            this.__expect(
+                this.inverted ? !success : success,
+                () =>
+                    `Expected _${valueToString(this.target)}_` +
+                    (this.inverted ? "not to eval but it did" : "to eval but it didn't")
+            );
```

Comment:
While writing the tests I hit these a lot, should this change also be tested? It is just human friendly output

Reply:
nope, that's ok. Thanks for making them more useful though!